.gitignore

```diff
@@ -21,4 +21,4 @@ docs/_static/Plotting_*.png
 docs/_static/Tutorial_*.png
 docs/generated
 poetry.lock
-references
+references
```

Comment:
**Minor**

Noeeeeeeeezzz :(

Reply:
No idea why this happened. :/ but I will revert before merge. So gross. I hate it too 😃